Nav.php

```diff
@@ -66,6 +67,10 @@ class Nav extends Widget
      */
     public $items = [];
     /**
+     * @var boolean whether the nav subitems should be a [[Dropdown]] widget.
+     */
+    public $asDropdown = true;
```

Comment:
I am unsure about name for this option.
Curently it does not sound  very good. Must be something consistent with `encodeLabels`, but I can't find a good name.